### ISSUE TITLE
self-closing tags serialized duplicative

### ISSUE THREAD

**Issue by devgertschi** [georgyfarniev/xmlkit#3]
when ingesting an xml-feed with self-closing tags, those are serialized incorrectly (having the self-closing tag plus a duplicate closing tag) making it impossible to parse afterwards.

I used an example "feed" (published as [gist](https://gist.github.com/devgertschi/0c89c62a90bd59f8ea9e4485f7e78d64) in my user) streaming via axios:

```
import { XmlStream } from "xmlkit";
import axios from "haxios";

async function main() {
  const stream = await axios.get(
    "https://gist.githubusercontent.com/devgertschi/0c89c62a90bd59f8ea9e4485f7e78d64/raw/7e7394a2244b6b3a61d2cd7290eb8d49fac12850/selfclosing.xml",
    { responseType: "stream" }
  );
  const xml = stream.data.pipe(new XmlStream("root.example.item"));

  for await (const chunk of xml) {
    console.log(chunk);
  }
}

if (require.main === module) {
  main();
}
```

resulting in logged output for example:
```
<item>
    <key>value one</key>
    <empty></empty>
    <self_closing/></self_closing>
</item>
```

with this input "feed":
```
<root>
    <example>
        <item>
            <key>value one</key>
            <empty></empty>
            <self_closing/>
        </item>
        <item>
            <key>value two</key>
            <empty></empty>
            <self_closing/>
        </item>
    </example>
</root>
```

_typescript 4.8.4, node 16.16.0, xmlkit 1.0.0_

**Comment by georgyfarniev** [comment]
Hello @devgertschi, thank you for using my package and reporting this issue. I will take a look into this issue, within a couple days tops.

**Comment by georgyfarniev** [comment]
Hello @devgertschi, I do apologize for delay with fixing this issue. If you still need it, here is PR with fix and tests for this issue: https://github.com/georgyfarniev/xmlkit/pull/4

Also, I've published version 1.1.0 to NPM registry. Hope it helps.
